### DISCUSSION TITLE
0.15.3 packages unavailable

### DISCUSSION THREAD

**Discussion by obfuscurity** [go-graphite/carbonapi#612] [General]
Looks like the CI build for 0.15.3 failed?

https://travis-ci.org/github/go-graphite/carbonapi/builds/772088516

**Comment by deniszh** [comment]
```
Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```
:(
Looks like we need to migrate to https://ghcr.io, @Civil

**Comment by adamcstephens** [comment]
This looks like you're hitting the docker hub limit. https://docs.docker.com/docker-hub/download-rate-limit/

**Comment by deniszh** [comment]
Ah, looks like https://ghcr.io is commercial too, shucks.... we can use it for build, though, traffic from Github Actions is free.
